torchio/data/images.py

```diff
@@ -87,10 +87,11 @@ class Subject(list):
         **kwargs: Items that will be added to the subject sample.
     """
 
-    def __init__(self, *images: Image, **kwargs):
+    def __init__(self, *images: Image, name: str = '', **kwargs):
```

Comment:
Can you explain why you're keeping `name`?

Reply:
I guess you mean it's used in `Subject.__repr__`. That's not very important, so we can just get rid of it.